src/amo/components/EditableCollectionAddon/styles.scss

```diff
@@ -88,7 +88,7 @@ div.EditableCollectionAddon-notes {
   grid-column: 1 / 3;
   margin-top: $padding-page;
   padding: 12px;
-
+  word-break: break-all;
```

Comment:
Could you please add a blank line below?

Reply:
@willdurand I had to remove it to remove eslint error.

Reply:
@willdurand you can see, it's failing again -_-

Reply:
Yes but it's not because of the blank line:

> ✖  Unexpected whitespace at end of line   no-eol-whitespace

Please run `yarn prettier-dev`